_sass/_theme.scss

```diff
@@ -59,11 +59,11 @@ body {
 .logo {
   display: inline-block;
   line-height: 0;
-  max-height: 4rem;
+  height: 4rem;
```

Comment:
Could this fix be in it's own pull request? It's a better way to track fixes. Also, might be better to style the image so the size isn't a fixed width? Some users may upload quite a wide but short image, in which this case it will never be bigger than `4rem`